src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrebid.java

```diff
@@ -1,19 +1,32 @@
 package com.wikia.webdriver.testcases.adstests;
 
+import com.thoughtworks.selenium.Wait;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.thoughtworks.selenium.Wait'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)